source/river/assets/test-config.toml

```diff
@@ -0,0 +1,64 @@
+# Test configuration file
+#
+# This is more in depth than the example.
+#
+# NOTE: Until we have made a release, configuration format should
+# be considered entirely unstable, and likely to make breaking changes
+# commit-to-commit!
+
+[system]
+# Threads per service
+threads-per-service = 8
+
+# Specify a Basic Proxy service, which features minimal configuration.
+#
+# Note that indentation isn't significant in TOML, it is only done
+# here to note the structure of the data.
+#
+# TODO: This data is very structured, and TOML might not be a good fit.
+[[basic-proxy]]
+name = "Example1"
+
+    # Each `basic-proxy` can have one or more Listeners, or downstream
+    # connections. If you provide zero, the basic proxy will terminate
+    # immediately.
+    [[basic-proxy.listeners]]
+        # TODO: "listeners" only has one field, we might want to use
+        # serde flatten
+        [basic-proxy.listeners.source]
+        # Listeners can have kind of "Tcp" (w/ or w/o TLS) or "Uds"
+        # for "Unix Domain Sockets", which cannot have TLS.
```

Comment:
It might not be implemented in v1 but openssl s_client definitively works with `-unix` path.